packages/tablecloth-melange/tablecloth-melange.0.0.9/opam

```diff
@@ -16,7 +16,7 @@ depends: [
   "ocaml" {>= "5.1"}
   "dune" {>= "3.8"}
   "reason"
-  "melange"
+  "melange" {>= "2.0.0" & < "3.0.0"}
```

Comment:
I assume the lower bound is intentional and was incorrect here, but I prefer always to double check :)

Reply:
@anmonteiro please let me know and then I'll merge

Reply:
oops, not intentional. if this was tested with melange 1.0 then we can revert.